### PR TITLE
blockfrost Get current epoch information

### PR DESCRIPTION
**Pull request by Luis-omega**
Part of #1365 

### Pre-review checklist

- [x] All code has been formatted using our config (`make format`)
- [x] Any new API features or modification of existing behavior is covered as defined in our [test plan](https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/doc/test-plan.md)
- [x] The changelog has been updated under the `## Unreleased` header, using the appropriate sub-headings (`### Added`, `### Removed`, `### Fixed`), and the links to the appropriate issues/PRs have been included

### CODE REVIEW

**Comment by jy14898** [comment]
I'd like to take over this PR, given Luis is finishing work on CTL

**Review by errfrom — CHANGES_REQUESTED** [review]
LGTM, but please update `spago-packages.nix` of the template and extract `ProtocolParameters` into a separate module

File: templates/ctl-scaffold/packages.dhall [hunk 1]
```diff
@@ -348,7 +349,7 @@ let additions =
           , "variant"
           ]
         , repo = "https://github.com/Plutonomicon/cardano-transaction-lib.git"
-        , version = "001b639606f341489968d599fb0cef2900aeb474"
+        , version = "90b8fc187d61e6e3b91290339dc8039190cbfece"
```
[the remark this answers is not in the document]
Comment (jy14898) [reply templates/ctl-scaffold/packages.dhall:352]: I think the generated file already contains all the needed dependencies, we moved a transitive dep to a direct one. I'll check though

[the remark this answers is not in the document]
Comment (jy14898) [reply templates/ctl-scaffold/packages.dhall:352]: Oh, I see

**Review by errfrom — APPROVED** [review]

**Pull request merged**